src/App.jsx

```diff
@@ -1,25 +1,14 @@
+import { LandingPage } from "pages/";
+import { Footer, Navbar } from "components";
+
 function App() {
```

Comment:
You can change this as well to es6 arrow function as all your components are in arrow function syntax